x/ecocredit/simulation/genesis.go

```diff
@@ -171,50 +196,68 @@ func genGenesisState(ctx context.Context, r *rand.Rand, simState *module.Simulat
 
 	// create few classes
 	cKey1, err := ss.ClassInfoTable().InsertReturningID(ctx,
-		&api.ClassInfo{Id: "C01", Admin: accs[0].Address.Bytes(), Metadata: simtypes.RandStringOfLength(r, simtypes.RandIntBetween(r, 5, 100))},
+		&api.ClassInfo{
+			Id:               "C01",
+			Admin:            accs[0].Address,
+			Metadata:         simtypes.RandStringOfLength(r, simtypes.RandIntBetween(r, 5, 100)),
+			CreditTypeAbbrev: "C",
+		},
 	)
 	if err != nil {
 		return err
 	}
 
 	cKey2, err := ss.ClassInfoTable().InsertReturningID(ctx,
-		&api.ClassInfo{Id: "C02", Admin: accs[1].Address.Bytes(), Metadata: metadata},
+		&api.ClassInfo{
+			Id:               "C02",
+			Admin:            accs[1].Address,
+			Metadata:         metadata,
+			CreditTypeAbbrev: "C",
+		},
 	)
 	if err != nil {
 		return err
 	}
 
+	// class sequence
+	if err := ss.ClassSequenceTable().Save(ctx, &api.ClassSequence{
+		CreditTypeAbbrev: "C",
+		NextSequence:     cKey2 + 1,
+	}); err != nil {
+		return err
+	}
+
 	// create class issuers
 	if err := ss.ClassIssuerTable().Save(ctx,
-		&api.ClassIssuer{ClassKey: cKey1, Issuer: accs[0].Address.Bytes()},
+		&api.ClassIssuer{ClassKey: cKey1, Issuer: accs[0].Address},
 	); err != nil {
 		return err
 	}
 
 	if err := ss.ClassIssuerTable().Save(ctx,
-		&api.ClassIssuer{ClassKey: cKey1, Issuer: accs[1].Address.Bytes()},
+		&api.ClassIssuer{ClassKey: cKey1, Issuer: accs[1].Address},
 	); err != nil {
 		return err
 	}
 
 	if err := ss.ClassIssuerTable().Save(ctx,
-		&api.ClassIssuer{ClassKey: cKey2, Issuer: accs[1].Address.Bytes()},
+		&api.ClassIssuer{ClassKey: cKey2, Issuer: accs[1].Address},
 	); err != nil {
 		return err
 	}
 
 	if err := ss.ClassIssuerTable().Save(ctx,
-		&api.ClassIssuer{ClassKey: cKey2, Issuer: accs[2].Address.Bytes()},
+		&api.ClassIssuer{ClassKey: cKey2, Issuer: accs[2].Address},
 	); err != nil {
 		return err
 	}
 
 	// create few projects
 	pKey1, err := ss.ProjectInfoTable().InsertReturningID(ctx,
 		&api.ProjectInfo{
-			Key:                 cKey1,
-			Id:                  "P01",
-			Admin:               accs[0].Address.Bytes(),
+			ClassKey:            cKey1,
+			Id:                  "Project1",
```

Comment:
Ok to leave as is but we might be updating the format to only allow capital letters #1046